src/pages/patientView/timeline2/timeline_helpers.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import { allResultValuesAreNumerical } from 'pages/patientView/timeline2/timeline_helpers';
+import { assert } from 'chai';
+import { TimelineEvent } from 'cbioportal-clinical-timeline';
+
+describe('#allResultValuesAreNumerical', () => {
+    var events = [] as TimelineEvent[];
+
+    beforeEach(() => {
+        events = [
+            {
+                event: {
+                    attributes: [{ key: 'RESULT', value: '0.6' }],
+                },
+            },
+            {
+                event: {
+                    attributes: [{ key: 'RESULT', value: '10' }],
+                },
+            },
+        ] as TimelineEvent[];
+    });
+
+    it('catches all numerical condition', () => {
+        assert.isTrue(allResultValuesAreNumerical(events));
+    });
+
+    it('catches non numerical', () => {
+        events.push({
```

Comment:
this makes test run order brittle. how about `const events2 = [...events, { blah }];`

Reply:
no, events are refreshed in beforeEach

Reply:
oh oops